heron/scheduler-core/src/java/com/twitter/heron/scheduler/RuntimeManagerRunner.java

```diff
@@ -190,6 +210,15 @@ void updateTopologyHandler(String topologyName, String newParallelism)
       throw new UpdateDryRunResponse(topology, config, newPlan, oldPlan, changeRequests);
     }
 
+    int newContainerCount = proposedPlan.getContainerPlansCount();
+    int oldContainerCount = currentPlan.getContainerPlansCount();
+    if (newContainerCount > oldContainerCount) {
+      if (!consolePrompt(oldContainerCount, newContainerCount)) {
```

Comment:
I would suggest having an option (can be overridden by `config-override`) to bypass the prompt so it is easier to invoke the `update` command in other scripts or process, like ci, Jenkins.

Reply:
will do in another pr

Reply:
added the config to pass ci